GccBoardProject1/src/peopleCounting.h

```diff
@@ -8,17 +8,20 @@
 #ifndef PEOPLECOUNTING_H_
 #define PEOPLECOUNTING_H_
 
-#define COUNT_BUFF_SIZE 7
+#define COUNT_BUFF_SIZE 448
+#define COUNT_BUFF_FRAMES 7
 #define TRIGGER_COLUNM_2 2
 #define TRIGGER_COLUNM_5 5
 #define CHECK_OFFSET_2 1
-#define CHECk_OFFSET_5 -1
+#define CHECK_OFFSET_5 -1
 #define IN false
 #define OUT true
 #define TRIGGER_INDEX COUNT_BUFF_SIZE/2
-#define GRID_SIZE 8	
+#define GRID_SIZE 8
 #define TRIGGER_THRESHOLD 20
-#define MAX_THRESHOLD 10	
+#define MAX_THRESHOLD 10
+#define MEDIAN_FILTER_POINTERS_BUFF_SIZE 10
+#define NUM_RAW_FRAMES 10
 
 
 struct keepCount{
```

Comment:
I like the idea of making a static struct to better encapsulate all the things to do with people counting. That way, we can just pass the struct into each of the people counting functions internally and better namespace the bunches of static arrays we have. Then again, since we only ever need 1 people count to occur at once, this suggestion isn't super necessary, but my software (over)engineering senses are tingling.